CHANGELOG.md

```diff
@@ -1,10 +1,8 @@
 # Changelog
 
-## 1.1.1 (May 24, 2017)
+## 2.0.1 (Jan 16, 2017)
```

Comment:
Not sure what improvements may have been added in May 2017. @wenderjean please check.